Uncaught Exception: Cannot find module 'electron-context-menu'

### What happened?

Install Desktop/Android APP Nightly Release 20220216 over 0.5.9
Error occours while installing and every time i try running Logseq.
Deleted all Folders in %USERPROFILE%. Fresh install. Error occours in fresh install and when installed again. Installer seems broken for me.



### Reproduce the Bug

1. Install Nightly Build on windows. Doesn't care whether fresh install or udating 0.5.9
2. Error occurs while installing.
3. Error persists while trying to run.

### Expected Behavior

Installer running without error.
Logseq starting.

### Screenshots


### Desktop Platform Information

AMD Ryzen 7 5800H with Radeon Graphics            3.20 GHz
RAM	32,0 GB
System	64-Bit-OS, x64-based Processor

Edition	Windows 11 Home
Version	21H2
Installed on ‎07.‎01.‎2022
Build 22000.493

### Mobile Platform Information

_No response_

### Additional Context

_No response
![16-02-2022_22-50-24](https://user-images.githubusercontent.com/54500911/154365255-9ff99a5c-592a-4d54-9789-6334cbf6eb65.png)
_

Same behaviour on Mac12.3 M1
```
Uncaught Exception:
Error: Cannot find module 'electron-context-menu'
Require stack:
- /Applications/Logseq.app/Contents/Resources/app/electron.js
- 
at Module._resolveFilename (node:internal/modules/cjs/loader:934:15)
at Function.n._resolveFilename (node:electron/js2c/browser_init:249:1128)
at Module._load (node:internal/modules/cjs/loader:779:27)
at Function.c._load (node:electron/js2c/asar_bundle:5:13331)
at Module.require (node:internal/modules/cjs/loader:1006:19)
at require (node:internal/modules/cjs/helpers:93:18)
at /Applications/Logseq.app/Contents/Resources/app/electron.js:13994:55
at Object.<anonymous> (/Applications/Logseq.app/Contents/Resources/app/electron.js:14329:3)
at Module._compile (node:internal/modules/cjs/loader:1110:14)
at Object.Module._extensions..js (node:internal/modules/cjs/loader:1140:10)
```

--- a/package.json
+++ b/package.json
@@ -96,7 +96,6 @@
         "is-svg": "4.3.0",
         "jszip": "3.5.0",
         "mldoc": "1.3.0",
-        "remove-accents": "0.4.2",
         "path": "0.12.7",
         "pixi-graph-fork": "0.2.0",
         "pixi.js": "6.2.0",

--- a/resources/package.json
+++ b/resources/package.json
@@ -22,6 +22,7 @@
     "better-sqlite3": "7.4.5",
     "chokidar": "^3.5.1",
     "dugite": "1.103.0",
+    "electron-context-menu": "3.1.1",
     "electron-log": "4.3.1",
     "electron-squirrel-startup": "1.0.0",
     "electron-window-state": "5.0.3",

